### ISSUE TITLE
Visualize the dataset

### ISSUE THREAD

**Issue by meltingCat** [nmi-lab/decolle-public#6]
hi, is there the way to visualize N-MNIST dataset?

**Comment by eneftci** [comment]
Yeah, check out the test scripts in the package torchneuromorphic, i.e. test_nmnist or view_nmnist.ipynb. The function 
plot_frames_imshow(frames, labels) should do it.
